distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Aggregate_Helper.enso

```diff
@@ -29,7 +29,7 @@ make_aggregate_column : DB_Table -> Aggregate_Column -> Text -> Dialect -> (Any
 make_aggregate_column table aggregate as dialect infer_return_type problem_builder =
     is_non_empty_selector v = v.is_nothing.not && v.not_empty
     simple_aggregate op_kind columns =
-        expression = SQL_Expression.Operation op_kind (columns.map .expression)
+        expression = SQL_Expression.Operation op_kind (columns.map c->c.expression)
```

Comment:
Is this the mentioned
> * Can't use `.` to access a private constructor field when mapping, have to use a lambda.
issue?

It looks like a possible bug, shall we report it to the engine team?

Reply:
Yes .function, _.function and c->c.function should all work